changelog/pending/20240819--cli-plugin--fix-code-generation-with-aliases-in-golang.yaml

```diff
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: cli/plugin
+  description: Fix code generation with aliases in golang
```

Comment:
This change seems unrelated, or am I missing something?  It should not be in this PR if so.

Reply:
there are two prs (stacked commits) Ill make it more clear in the future.  This PR should only look at the last commit

Reply:
You can set the base for the PR to the earlier PR, then the first commit won't show up in it.  I think that makes it easier to review.